src/main/kotlin/three/consulting/epoc/controller/TimeCategoryController.kt

```diff
@@ -0,0 +1,27 @@
+package three.consulting.epoc.controller
+
+import org.springframework.http.MediaType.ALL_VALUE
+import org.springframework.web.bind.annotation.*
+import three.consulting.epoc.dto.TimeCategoryDTO
+import three.consulting.epoc.service.TimeCategoryService
+import javax.validation.Valid
+
+@RestController
+@RequestMapping(path = ["/timeCategory"])
+class TimeCategoryController(private val timeCategoryService: TimeCategoryService) {
+
+    @GetMapping(value = ["/{timeCategoryId}"], consumes = [ALL_VALUE])
+    fun getTimeCategoryForId(@PathVariable timeCategoryId: Long) = timeCategoryService.findTimeCategoryForId(timeCategoryId)
+
+    @PostMapping
```

Comment:
Not an issue for this pr but since the GET is allowed to consume all values, the POST/PUT should be limited to APPLICATION_JSON or whatever it is. Similarly all controller methods should have the consumes/produces values specified. Just a thought.

Reply:
Ill look into this and make a separate PR for fixing all controller methods